maven-scm-api/src/main/java/org/apache/maven/scm/command/untag/AbstractUntagCommand.java

```diff
@@ -31,18 +31,27 @@
 public abstract class AbstractUntagCommand
     extends AbstractCommand
 {
+    /**
+     * SCM-917 - scm:untag for subversion
```

Comment:
This line is redundant, we have git log for.

Reply:
Will replace by 'execute untag command'